source/Playnite.Common/FileSystem.cs

```diff
@@ -325,5 +325,46 @@ public static void CopyDirectory(string sourceDirName, string destDirName, bool
                 }
             }
         }
+
+        public static string LookupAlternativeFilePath(string filePath, bool errorIfNotFullPath)
+        {
+            return CheckDrivesForPath(filePath, errorIfNotFullPath, path => File.Exists(path));
+        }
+
+        public static string LookupAlternativeDirectoryPath(string directoryPath, bool errorIfNotFullPath)
+        {
+            return CheckDrivesForPath(directoryPath, errorIfNotFullPath, path => Directory.Exists(path));
+        }
+
+        private static string CheckDrivesForPath(string originalPath, bool errorIfNotFullPath, Predicate<string> predicate)
+        {
+            if (!Paths.IsFullPath(originalPath))
+            {
+                if (errorIfNotFullPath)

```

Comment:
I really think we don't need to have an option to throw an error. Especially if we don't use it anywhere.